### PR TITLE
[WIP] Update version information for Python packaging of nteract_on_jupyter

### LINKED ISSUE #4469: Confusing metadata regarding the version of nteract-on-jupyter
I'm trying to package https://pypi.org/project/nteract-on-jupyter/2.0.12/ but while the PyPI version is 2.0.12 the source distribution provides 0.0.1-dev and the source here is 0.0.o-dev, see https://github.com/nteract/nteract/blob/54c50e856930e9489e9b920365dacc3f10320af4/applications/jupyter-extension/setup.py#L8

Not sure if the package on PyPI was originated from the code above BTW, this repo is quite big and hard to navigate, so feel free to close if I'm in the wrong place.

### PR DESCRIPTION
**Pull request by willingc**
- add LICENSE to manifest
- remove default to 0 version information
- pull version information from the package.json in nteract_on_jupyter directory
- update RELEASE information for nteract_on_jupyter for PyPI and conda-forge

Closes #4469

### CODE REVIEW

**Comment by willingc** [comment]
- [x] check PACKAGEDIR lands in nteract_on_jupyter or modify in setup.py here.
- [ ] check if [nbextension version](https://github.com/nteract/nteract/blob/master/applications/jupyter-extension/package.json#L3) in package.json should be the same as nteract_on_jupyter pip version.

**Comment by rgbkrk** [comment]
The `jupyter-extension` version (at `applications/jupyter-extension/package.json`) is irrelevant because it's not the thing we package up. It's more of an internal metapackage than anything else, that helped us work around python needing it's particular directory structure and the monorepo tools at the time requiring us to have this package.json here.

File: RELEASING.md [hunk 1]
```diff
@@ -71,3 +71,9 @@ babby
 ```
 
 5.  Once you're ready click "Publish release". On macOS and Windows, the update will be automatically downloaded and installed.
+
+## Releasing nteract web application
+
+To release **nteract_on_jupyter** web application on pip or conda-forge, please follow these steps:
+
+TODO
```
Comment (rgbkrk) [inline RELEASING.md:79]: 😄

File: applications/jupyter-extension/MANIFEST.in [hunk 2]
```diff
@@ -1,4 +1,5 @@
 include README.md
+include ../LICENSE
```
Comment (rgbkrk) [inline applications/jupyter-extension/MANIFEST.in:2]: 👍

File: applications/jupyter-extension/nteract_on_jupyter/_version.py [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
+
+__version__ and version_info are set using the version information
+contained in the package.json for this application.
+"""
 import json
 import os
 
-version_info = (0, 0, 0)
-__version__ = "0.0.0"
-
```
Comment (rgbkrk) [inline applications/jupyter-extension/nteract_on_jupyter/_version.py:6]: For some reason, I thought I had to have this as a placeholder. Guess not!

**Comment by captainsafia** [comment]
@rgbkrk @willingc Is there any work needed to get this merged? It looks like the second item in @willingc's checklist was addressed.

**Comment by rgbkrk** [comment]
We probably still need this, or we need to deprecate our nteract on Jupyter app entirely since we're not maintaining it/developing it.

**Comment by captainsafia** [comment]
The plan per https://github.com/nteract/nteract/issues/4956 is to eventually deprecate the Jupyter extension in favor of the web front-end.

**Pull request closed without merging**